docs/kubernetes/routegroups.md

```diff
@@ -0,0 +1,628 @@
+
+## Usage
+
+The absolute minimal route group configuration for a Kubernetes service (my-service) looks as follows:
+
+```yaml
+apiVersion: zalando.org/v1
+kind: RouteGroup
+metadata:
+  name: my-route-group
+spec:
+  backends:
+  - name: my-backend
+    type: service
+    serviceName: my-service
+    servicePort: 80
+  defaultBackends:
+  - backendName: my-service
+```
+
+This is equivalent to the ingress:
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: my-ingress
+spec:
+  backend:
+    serviceName: my-service
+    servicePort: 80
+```
+
+Notice that the route group contains a list of actual backends, and the defined service backend is then
+referenced as the default backend. This structure plays a role in supporting scenarios like A/B testing and
+gradual traffic switching, [explained below](#gradual-traffic-switching). The backend definition also has a type
+field, whose values can be service, lb, network, shunt, loopback or dynamic. More details on that
+[below](#backends).
+
+Creating, updating and deleting route groups happens the same way as with ingress objects. E.g, manually
+applying a route group definition:
+
+```
+kubectl apply -f my-route-group.yaml
+```
+
+## Hosts
+
+- *[Format](../routegroup-crd/#routegroup-top-level-object)*
+
+Hosts contain hostnames that are used to match the requests handled by a given route group. They are also used
+to update the required DNS entries and load balancer configuration if the cluster is set up that way.
+
+Note that it is also possible to use any Skipper predicate in the routes of a route group, with the Host
+predicate included, but the hostnames defined that way will not serve as input for the DNS configuration.
+
+## Backends
+
+- *[Format](../routegroup-crd/#backend_1)*
+- *[General backend reference](../../reference/backends/)*
+
+RouteGroups support different backends. The most typical backend type is the 'service', and it works the same
+way as in case of ingress definitions.
+
+In a RouteGroup, there can be multiple backends and they are listed on the top level of the route group spec,
+and are referenced from the actual routes or as default backends.
+
+### type=service
+
+This backend resolves to a Kubernetes service. It works the same way as in case of Ingress definitions. Skipper
+resolves the Services to the available Endpoints belonging to the Service, and generates load balanced routes
+using them. (This basically means that under the hood, a `service` backend becomes an `lb` backend.)
+
+### type=lb
+
+This backend provides load balancing between multiple network endpoints. Keep in mind that the service type
+backend automatically generates load balanced routes for the service endpoints, so this backend type typically
+doesn't need to be used for services.
+
+### type=network
+
+This backend type results in routes that forward incoming requests to the defined network address, regardless of
```

Comment:
forward -> proxy

The problem is that forward is not equal to proxy. As an example, think about NLB. NLB is a packet forwarder, such that the client IP is the real IP, not the NLB. This would be miss leading in the context for people that know more network or load balancers